src/modules/comments.js

```diff
@@ -0,0 +1,98 @@
+import { sendData, getData, counter } from './AddCommentAPI.js';
+
+const getCommentData = async (idMeals) => {
+  const request = await fetch(`https://themealdb.com/api/json/v1/1/lookup.php?i=${idMeals}`);
+  const response = await request.json();
+  return response;
+};
+
+async function openPopup(data) {
+  const container = document.querySelector('.comments');
+  const popup = `<div class="outside">
+    <p class="closeBtn">X</p>
+    <img src=${data.strMealThumb} class="API-img">
+    <p class="image-title">${data.strMeal}</p>
+    <div class="category&type>
+      <p>${data.strArea}</p>
+      <p>${data.strCategory}</p>
+    </div>
```

Comment:
- [ ] Please, It's one of the project requirements you use ES6, let's use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)